x/magpie/internal/keeper/keeper.go

```diff
@@ -20,6 +20,29 @@ func NewKeeper(cdc *codec.Codec, storeKey sdk.StoreKey) Keeper {
 	}
 }
 
+// -------------
+// --- Params
+// -------------
+
+// SetDefaultSessionLength allows to set a default session length for new magpie sessions.
+// The specified length is intended to be in number of blocks.
+func (k Keeper) SetDefaultSessionLength(ctx sdk.Context, length int64) {
```

Comment:
Is here a specific reason why you choose `int64` instead of `uint64`?

Reply:
By using `int64` the method usage is simplified inside the `handler` (which is the only meaningful place where it's used) as `ctx.BlockHeight()` is an `int64`. 

Also, if we would have to use `uint64` we should perform additional checks later as we could set the default session length to a value up to `2^64`, which would not be supported inside the `expiration` field of the `Session` type (which is a `int64` that allows values up to `2^32`)

Reply:
That's make sense!